{"uuid":"957a3870-ea48-4784-8819-dfd3678e45e4","historyId":"42fae34c-44d0-59ce-a91a-8e83ce27121f","status":"skipped","statusDetails":{"message":"`cy.request()` failed on:\n\nhttps://amei-homolog.amorsaude.com.br/api/v1/security/login\n\nThe response we received from your web server was:\n\n  > 401: Unauthorized\n\nThis was considered a failure because the status code was not `2xx` or `3xx`.\n\nIf you do not want status codes to cause failures pass the option: `failOnStatusCode: false`\n\n-----------------------------------------------------------\n\nThe request we sent was:\n\nMethod: POST\nURL: https://amei-homolog.amorsaude.com.br/api/v1/security/login\nHeaders: {\n  \"Connection\": \"keep-alive\",\n  \"user-agent\": \"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/139.0.0.0 Safari/537.36 Edg/139.0.0.0\",\n  \"accept\": \"*/*\",\n  \"accept-encoding\": \"gzip, deflate\",\n  \"content-type\": \"application/json\",\n  \"content-length\": 63\n}\nBody: {\"email\":\"ivan.santos+1@amorsaude.com\",\"password\":\"Iv@n198529\"}\n\n-----------------------------------------------------------\n\nThe response we got was:\n\nStatus: 401 - Unauthorized\nHeaders: {\n  \"content-type\": \"application/json; charset=utf-8\",\n  \"content-length\": \"144\",\n  \"connection\": \"keep-alive\",\n  \"x-powered-by\": \"Express\",\n  \"vary\": \"Origin\",\n  \"x-ratelimit-limit\": \"150\",\n  \"x-ratelimit-remaining\": \"148\",\n  \"x-ratelimit-reset\": \"52\",\n  \"etag\": \"W/\\\"90-kMDepBIYndL/Qehj3uLTv+N2OQM\\\"\",\n  \"date\": \"Mon, 11 Aug 2025 14:16:09 GMT\",\n  \"x-cache\": \"Error from cloudfront\",\n  \"via\": \"1.1 e077d37d87018bf679a956b0f7e932ae.cloudfront.net (CloudFront)\",\n  \"x-amz-cf-pop\": \"GRU3-P5\",\n  \"x-amz-cf-id\": \"03_TCqUKbqTCneFF9kCN6O4G4bVgot6kIQcW29odv_wMUXqPu0tq1A==\"\n}\nBody: {\n  \"message\": \"ORA-12514: TNS:listener does not currently know of service requested in connect descriptor\",\n  \"error\": \"Unauthorized\",\n  \"statusCode\": 401\n}\n\n\nBecause this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `Módulo - Origens`","trace":"CypressError: `cy.request()` failed on:\n\nhttps://amei-homolog.amorsaude.com.br/api/v1/security/login\n\nThe response we received from your web server was:\n\n  > 401: Unauthorized\n\nThis was considered a failure because the status code was not `2xx` or `3xx`.\n\nIf you do not want status codes to cause failures pass the option: `failOnStatusCode: false`\n\n-----------------------------------------------------------\n\nThe request we sent was:\n\nMethod: POST\nURL: https://amei-homolog.amorsaude.com.br/api/v1/security/login\nHeaders: {\n  \"Connection\": \"keep-alive\",\n  \"user-agent\": \"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/139.0.0.0 Safari/537.36 Edg/139.0.0.0\",\n  \"accept\": \"*/*\",\n  \"accept-encoding\": \"gzip, deflate\",\n  \"content-type\": \"application/json\",\n  \"content-length\": 63\n}\nBody: {\"email\":\"ivan.santos+1@amorsaude.com\",\"password\":\"Iv@n198529\"}\n\n-----------------------------------------------------------\n\nThe response we got was:\n\nStatus: 401 - Unauthorized\nHeaders: {\n  \"content-type\": \"application/json; charset=utf-8\",\n  \"content-length\": \"144\",\n  \"connection\": \"keep-alive\",\n  \"x-powered-by\": \"Express\",\n  \"vary\": \"Origin\",\n  \"x-ratelimit-limit\": \"150\",\n  \"x-ratelimit-remaining\": \"148\",\n  \"x-ratelimit-reset\": \"52\",\n  \"etag\": \"W/\\\"90-kMDepBIYndL/Qehj3uLTv+N2OQM\\\"\",\n  \"date\": \"Mon, 11 Aug 2025 14:16:09 GMT\",\n  \"x-cache\": \"Error from cloudfront\",\n  \"via\": \"1.1 e077d37d87018bf679a956b0f7e932ae.cloudfront.net (CloudFront)\",\n  \"x-amz-cf-pop\": \"GRU3-P5\",\n  \"x-amz-cf-id\": \"03_TCqUKbqTCneFF9kCN6O4G4bVgot6kIQcW29odv_wMUXqPu0tq1A==\"\n}\nBody: {\n  \"message\": \"ORA-12514: TNS:listener does not currently know of service requested in connect descriptor\",\n  \"error\": \"Unauthorized\",\n  \"statusCode\": 401\n}\n\n\nBecause this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `Módulo - Origens`\n    at <unknown> (https://amei-homolog.amorsaude.com.br/__cypress/runner/cypress_runner.js:142607:80)\nFrom previous event:\n    at Context.request (https://amei-homolog.amorsaude.com.br/__cypress/runner/cypress_runner.js:142602:35)\n    at wrapped (https://amei-homolog.amorsaude.com.br/__cypress/runner/cypress_runner.js:153994:43)\nFrom Your Spec Code:\n    at Context.eval (webpack:///./cypress/support/commands.js:3:12)"},"stage":"pending","steps":[],"attachments":[],"parameters":[],"labels":[{"name":"package","value":"cypress\\e2e\\AWS\\origens.cy.js"},{"name":"path","value":"cypress\\e2e\\AWS\\origens.cy.js"},{"name":"parentSuite","value":"Módulo - Origens"},{"name":"suite","value":"Módulo - Origens - Apagar uma origem"}],"links":[],"start":1754921768518,"name":"Validar retorno 404 - /api/v1/origins","fullName":"Módulo - Origens Módulo - Origens - Listar Origens Validar retorno 404 - /api/v1/origins","stop":1754921768519}